### PR TITLE
Update regen/feature.pl

### PR DESCRIPTION
**Pull request by atoomic**
Features bundles are built on top of each other.

This change make it easier to read what features
were added from one version to another.
It also helps identify similar bundles.

Features are also now sorted on pod.

Enforce strict & warnings to generate feature.pm

### CODE REVIEW

**Comment by leonerd** [comment]
This seems like a bad idea. We're about to start *removing* some of the features as well (e.g. `indirect`), which this mechanism wouldn't be able to cope with.

**Comment by atoomic** [comment]
@leonerd note that you can either do `@new_bundle = qw{ full list... }` or `@new_bundles = map { $_ !~ qr{^(?:indirect|unicode_strings)$} } @previous_bundle`

I do not see how a simplified syntax for the past can impact the future.

**Comment by leonerd** [comment]
Ah hmmm, yes I guess that's true.

**Comment by atoomic** [comment]
@leonerd let me know if we should hold that Pull Request or if you are fine with it going in now?

**Comment by leonerd** [comment]
Go for it - mine isn't ready yet anyway

File: regen/feature.pl [hunk 1]
```diff
@@ -44,33 +45,37 @@ BEGIN
...
-    "5.29"   =>	[qw(say state switch unicode_strings unicode_eval
-		    evalbytes current_sub fc postderef_qq bitwise indirect)],
-    "5.31"   =>	[qw(say state switch unicode_strings unicode_eval
-		    evalbytes current_sub fc postderef_qq bitwise indirect)],
-    "5.33"   =>	[qw(say state switch unicode_strings unicode_eval
-		    evalbytes current_sub fc postderef_qq bitwise indirect)],
+    all     => [@all_features],
+    default => [qw(indirect)],
+    # using 5.9.5 features bundle
+    "5.9.5" => [@fb_5_9_5],
```
Comment (toddr) [inline regen/feature.pl]: could also be:
`"5.9.5" => \@fb_5_9_5,`

is there a risk doing it that way?

Comment (atoomic) [reply regen/feature.pl]: nope that should be fine, I also considered using it... 
but at the end this is a micro performance win for a regen script...

Comment (toddr) [reply regen/feature.pl]: apologies. I keep looking at this thinking I'm looking at feature.pm

File: lib/feature.pm [hunk 2]
```diff
@@ -388,51 +388,47 @@ The following feature bundles are available:
   --------- -----------------
   :default  indirect
 
-  :5.10     say state switch indirect
+  :5.10     indirect say state switch
```
Comment (toddr) [inline lib/feature.pm:391]: I realize this is only a POD change but it's a big re-sort. What is the new sorting algorithm?

Comment (atoomic) [reply lib/feature.pm:391]: it's now using an alphabetical sort making it easier to read and parse

**Pull request merged**